Add Bot API Version Status Badge

kamikazechaser opened the pull request:
**Ref:** #260 
**Info:** Displays the Bot API version as a status badge.

GochoMugo commented on README.md:
```diff
@@ -1,3 +1,4 @@
+[![Bot API](http://img.shields.io/badge/Bot API-v2.3.1-00aced.svg?style=flat-square)](https://github.com/yagop/node-telegram-bot-api)
 [![Build Status](https://travis-ci.org/yagop/node-telegram-bot-api.svg?branch=master)](https://travis-ci.org/yagop/node-telegram-bot-api)
```
indent this by one space, please!

GochoMugo commented on README.md:
```diff
@@ -1,3 +1,4 @@
+[![Bot API](http://img.shields.io/badge/Bot API-v2.3.1-00aced.svg?style=flat-square)](https://github.com/yagop/node-telegram-bot-api)
```
I think this should point to https://core.telegram.org/bots/api

GochoMugo commented on README.md:
```diff
@@ -1,5 +1,5 @@
-[![Bot API](http://img.shields.io/badge/Bot API-v2.3.1-00aced.svg?style=flat-square)](https://github.com/yagop/node-telegram-bot-api)
-[![Build Status](https://travis-ci.org/yagop/node-telegram-bot-api.svg?branch=master)](https://travis-ci.org/yagop/node-telegram-bot-api)
+ [![Bot API](http://img.shields.io/badge/Bot API-v2.3.1-00aced.svg?style=flat-square)](https://core.telegram.org/bots/api)
```
sorry, the 1st line is **not** indented!

GochoMugo commented on README.md:
```diff
@@ -1,5 +1,5 @@
-[![Bot API](http://img.shields.io/badge/Bot API-v2.3.1-00aced.svg?style=flat-square)](https://github.com/yagop/node-telegram-bot-api)
-[![Build Status](https://travis-ci.org/yagop/node-telegram-bot-api.svg?branch=master)](https://travis-ci.org/yagop/node-telegram-bot-api)
+ [![Bot API](http://img.shields.io/badge/Bot API-v2.3.1-00aced.svg?style=flat-square)](https://core.telegram.org/bots/api)
+ [![Build Status](https://travis-ci.org/yagop/node-telegram-bot-api.svg?branch=master)](https://travis-ci.org/yagop/node-  telegram-bot-api)
```
is there some space after `node-` in the link?

kamikazechaser commented:
Fixed [indentation](https://raw.githubusercontent.com/kamikazechaser/node-telegram-bot-api/67c16adcd0f10b772c3a17355f0d3b253b5a7c84/README.md) and changed [re-direct url](https://core.telegram.org/bots/api)
